### ISSUE TITLE
Chat messages are not accessible via keyboard navigation

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/spreed#3986]
## Steps to reproduce
1. Open a Talk conversation
2. Use only the keyboard with Tab-navigation and enter
3. Try to go to a link in a message or use the reply button (e.g. try shift-Tab from inside the new message input field)

### Expected behaviour
It works just fine to navigate between messages, replying as well as going to links and opening files which were sent.

### Actual behaviour
It is impossible to navigate the message list via keyboard. Shift-Tab from the message field doesn’t go further up than the attachment button.

(This is on our own instance so easy to test @ma12-co )

**Comment by PVince81** [comment]
For the message list we should probably use the arrow keys for navigating and tab/shift+tab to move to another field.
We shouldn't use tab for switching between messages as it would make it close to impossible to leave that box, or would take a while if there is scrolling involved and loading of old messages.

**Comment by PVince81** [comment]
In general I think we might want to introduce yet another highlight to be able to select a message. The selection would be done either by clicking or moving with the arrow keys. This also means that when coming from search we wouldn't simply flash/highlight the entry (like https://github.com/nextcloud/spreed/issues/4516) but select it.

Thoughts ?

**Comment by PVince81** [comment]
I've tested other solutions and it seems arrow keys is the way to go. The arrow keys do this when reaching a message, each entry below is the next one when hitting the down arrow key
- focus on author name
- focus on message or image
- focus on action buttons (like reply), if any
- next message

And using tab makes it possible to leave the messages list

**Comment by PVince81** [comment]
- [ ] some label on the converstion content so that when focussed it says "conversation X panel" or something

**Comment by PVince81** [comment]
some attempt here: https://github.com/nextcloud/spreed/pull/4666
